docs/api.md

```diff
@@ -0,0 +1,28 @@
+# API Reference
```

Comment:
I would add this:
```
---
hide:
  - navigation
---
```
which hides the left-hand side navigation which just takes unnecessary space in this page (have no sub-menus).